autoscaler/controllers/gateway/config/quickwit.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	odigosv1 "github.com/keyval-dev/odigos/api/odigos/v1alpha1"
+	commonconf "github.com/keyval-dev/odigos/autoscaler/controllers/common"
+	"github.com/keyval-dev/odigos/common"
+)
+
+const (
+	qwUrlKey = "QUICKWIT_URL"
+)
+
+type Quickwit struct{}
+
+func (e *Quickwit) DestType() common.DestinationType {
+	return common.QuickwitDestinationType
+}
+
+func (e *Quickwit) ModifyConfig(dest *odigosv1.Destination, currentConfig *commonconf.Config) {
+	if url, exists := dest.Spec.Data[qwUrlKey]; exists {
+		exporterName := "otlp/quickwit/" + dest.Name
```

Comment:
nit - all others exporters use this notation: 
```suggestion
		exporterName := "otlp/quickwit-" + dest.Name
```

Can you please update so this destination is aligned with the rest of the codebase? thanks

Reply:
Done